packages/lumx-react/src/components/link-preview/LinkPreview.tsx

```diff
@@ -62,13 +67,24 @@ const DEFAULT_PROPS: Partial<LinkPreviewProps> = {
  * @return React element.
  */
 export const LinkPreview: Comp<LinkPreviewProps, HTMLDivElement> = forwardRef((props, ref) => {
-    const { className, description, link, linkProps, size, theme, thumbnailProps, title, ...forwardedProps } = props;
-
-    //TODO: a11y
-    const goToUrl = useCallback(() => window.open(link, '_blank'), [link]);
+    const {
+        className,
+        description,
+        link,
+        linkAs,
+        linkProps,
+        size,
+        theme,
+        thumbnailProps,
+        title,
+        titleHeading,
+        ...forwardedProps
+    } = props;
+    // Use title heading as title wrapper (see DEFAULT_PROPS for the default value).
+    const TitleHeading = titleHeading as HeadingElement;
```

Comment:
Huh, was the `as` needed?
(Not an issue, just genuinely curious)

Reply:
Yeah it's needed otherwise the type is `HeadingElement | undefined` since he does not now that React will use the `defaultProps` if the prop is undefined and thus this prop cannot be `undefined`

Reply:
Ah I see, very well then :)